### ISSUE TITLE
Korjaa ei-merkitseville rakennuksille syntyvät erilliset kohteet

### ISSUE THREAD

**Issue by Rikuoja** [GispoCoding/jkr-core#37]
When creating kohteet from perusmaksut, it looks like in some cases, auxiliary buildings (e.g. autotallit) are not added to the kohde, so they are created as separate kohde instead. They should be added by [_add_auxiliary_buildings](https://github.com/GispoCoding/jkr-core/blob/main/jkrimporter/providers/db/services/kohde.py#L406) function.

Probably has to do with their address or owner. Anyway, at least prevent separate kohteet from being created from auxiliary buildings.

Basically, implement a function checking for auxiliary buildings (e.g. https://github.com/GispoCoding/jkr-lahti/issues/74) and use it to never create kohteet from such buildings.

**Comment by Rikuoja** [comment]
Fixed in https://github.com/GispoCoding/jkr-core/commit/96ae30dede0dad5e1e98b1134e8d8abeb1b64725
